flexbox-practice.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flexbox Practice Site</title>
+    <link rel="stylesheet" href="css-flexbox-practice.css">
+</head>
+<body>
+    <div class="site">
+        <nav class="navigation">
+            <img src="http://placehold.it/" alt="logo">
+            <ul>
+                <li><a href="#">about US</a></li>
+                <li><a href="#">Products</a></li>
+                <li><a href="#">Contacts</a></li>
+            </ul>           
+        </nav>
+        <div class="section-1">
+            <div class="section-1-left">        
+                <img src="http://placehold.it/" class="img" alt="image">
+            </div>
+            <div class="section-1-right">
+                <h2>Title</h2>
+                <p>Some Text Down Here</p>
+                <button>Button</button>
+                <button class="m">Button</button>
```

Comment:
Try to use a better classname then 'm'.
Classnames should be nouns, usually.
However, if they are meant to add some sort of direct css-styling (like margin, padding, width), try to use "margin", "padding", "width" in the class name...
example:
pad-small
margin-small
width-large
... etc.